frontend/libs/studio-components/src/components/StudioError/StudioError.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Alert } from '@digdir/designsystemet-react';
+
+export type StudioErrorProps = {
+  children?: React.ReactNode;
+};
+
+export const StudioError = ({ children }: StudioErrorProps) => {
+  return <Alert severity='danger'>{children}</Alert>;
+};
```

Comment:
Ref. previous comment, we must spread the rest of the props. 😊 

```suggestion
export const StudioError = ({ children, ...rest }: StudioErrorProps) => {
  return <Alert {...rest} severity='danger'>{children}</Alert>;
};
```